tests/fixtures/mock.py

```diff
@@ -18,3 +18,10 @@
 # functions, therefore the patching has no effect
 mp = monkeypatch()
 mp.setattr('django.utils.functional.cached_property', property)
+
+
+class FakeJob(object):
+    id = 'FAKE_JOB_ID'
+
+
+mp.setattr('rq.get_current_job', lambda: FakeJob())
```

Comment:
You don't want to use `mp.setattr()` here (the code above is an exception, reasoning is explained on the *HACKISH* part), but rather create a fixture which you can use in any test. Check out [this example](http://stackoverflow.com/a/20503374/783019) for reference.